### PR TITLE
Feature/story 22

### PR DESCRIPTION
**Pull request by Edfreded**
added [authorized] attributes on endpoints

### CODE REVIEW

**Comment by Edfreded** [comment]
Tests are failing because endpoint are now protected. the unit tests have to be rewritten

**Review by fjspaak — APPROVED** [review]

File: Tests/AllYourGoods.Api.IntegrationTests/GenerateJWT.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    //     var token = new JwtSecurityToken(
+    //         issuer: _configuration["Jwt:Issuer"],
+    //         audience: _configuration["Jwt:Audience"],
+    //         claims: claims,
+    //         expires: DateTime.Now.AddMinutes(5),
+    //         signingCredentials: creds
+    //     );
+
+    //     return new JwtSecurityTokenHandler().WriteToken(token);
+    // }
```
Comment (KiwiLT) [inline Tests/AllYourGoods.Api.IntegrationTests/GenerateJWT.cs:58]: Willen we deze comment er nog in houden?

File: Tests/AllYourGoods.Api.IntegrationTests/GenerateJWT.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        };
+
+        var key = new SymmetricSecurityKey(Encoding.UTF8.GetBytes("SuperSecretIdentityTestKeyThatIsLongEnoughNow"));
+        var creds = new SigningCredentials(key, SecurityAlgorithms.HmacSha256);
+
+        var token = new JwtSecurityToken(
+            issuer: "YourApp",
+            audience: "YourAppUsers",
+            claims: claims,
+            expires: DateTime.Now.AddMinutes(5),
```
Comment (KiwiLT) [inline Tests/AllYourGoods.Api.IntegrationTests/GenerateJWT.cs:25]: klopt JWT van 5 minuten?

**Review by KiwiLT — APPROVED** [review]
LGTM

File: Tests/AllYourGoods.Api.IntegrationTests/GenerateJWT.cs (hunk 2 above)
Comment (Edfreded) [reply Tests/AllYourGoods.Api.IntegrationTests/GenerateJWT.cs:25]: is now 30

**Pull request merged**